manifests/init.pp

```diff
@@ -4,15 +4,15 @@
 #
 # === Parameters:
 #
-# $enable_yum::                                Enable rpm content plugin, including syncing of yum content
+# $enable_yum::                                Enable rpm content plugin to sync and serve rpm content to be consumed by package manager like yum and dnf
```

Comment:
```suggestion
# $enable_yum::                                Enable the RPM content plugin to sync and serve rpm content to be consumed by package manager like yum and dnf
```

If we are going to refer to `content plugin` I think we should then also include that it is `Pulp RPM content plugin` to be specific here. Otherwise drop the `content plugin` and just refer to it as `Enable RPM content`

Reply:
I would also think about `Enable support for RPM content`

Reply:
> I would also think about `Enable support for RPM content`

I like this wording

Reply:
How about:

`Enable feature to sync, manage, and serve RPM content to be consumed by package managers like yum and dnf.`

`Enable feature to sync, manage, and serve file content.`

et cetera

IMO it's the best balance yet of being truly accurate (we're enabling the entire feature, not only the one piece of that feature which is the pulp plugin) and also very easily understood (you can sync, manage, and serve X content type)

Reply:
BTW I used `enable feature` instead of `enable support` because otherwise I promise there is SOMEBODY out there who is going to think it has to do with SLAs :)